clients/clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -222,18 +222,12 @@ interface ClearlyDefinedService {
     suspend fun searchDefinitions(@Query("pattern") pattern: String): List<String>
 
     /**
-     * Get the curation for the component described by [type], [provider], [namespace] (use "-" if not applicable),
-     * [name] and [revision], see
+     * Get the curation for the component described by the [coordinates]. When the [revision][Coordinates.revision] is
+     * omitted or empty, the latest revision is used (if that makes sense for the provider). Also see
      * https://api.clearlydefined.io/api-docs/#/curations/get_curations__type___provider___namespace___name___revision_.
      */
-    @GET("curations/{type}/{provider}/{namespace}/{name}/{revision}")
-    suspend fun getCuration(
-        @Path("type") type: ComponentType,
-        @Path("provider") provider: Provider,
-        @Path("namespace") namespace: String,
-        @Path("name") name: String,
-        @Path("revision") revision: String
-    ): Curation
+    @GET("curations/{coordinates}")
```

Comment:
I had to solve this differently as Retrofit escapes `/` contained in the string representation of `coordinates` as `%2F` instead of literally using it as a path.

Reply:
I actually [*did* find a way to solve the escaping problem](https://github.com/oss-review-toolkit/ort/pull/7688). And yes, we need to rely on the `toString()` implementation, as that's what Retrofit uses to turn objects into paths.